packages/components/src/components/expandable/expandable.stories.ts

```diff
@@ -30,222 +33,52 @@ export const Default = {
-        Background primary, inverted
-      </div>
-      <div class="background-sample bg-primary">
-        ${generateTemplate({
-          args: { ...args, inverted: true }
-        })}
-      </div>
-      <div class="w-full p-4 mb-8 bg-neutral-100 text-left text-[14px] font-bold box-border">Lead Text Example</div>
-      <div class="background-sample bg-neutral-100 ">
-        ${generateTemplate({
-          args: overrideArgs([
-            {
-              type: 'slot',
-              name: 'default',
-              value:
-                '<div class="sd-leadtext">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh justo ullamcorper odio tempor molestie phasellus dui vel id. Velit in sed non orci pellentesque vivamus nunc. At non tortor, sit neque tristique. Facilisis commodo integer hendrerit tortor. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh justo ullamcorper odio tempor molestie phasellus dui vel id. Velit in sed non orci pellentesque vivamus nunc. At non tortor, sit neque tristique. Facilisis commodo integer hendrerit tortor.</div>'
-            },
-            {
-              type: 'attribute',
-              name: 'variant',
-              value: 'leadtext'
-            },
-            {
-              type: 'cssProperty',
-              name: '--gradient-color-start',
-              value: 'rgba(246, 246, 246, 0)'
-            },
-            {
-              type: 'cssProperty',
-              name: '--gradient-color-end',
-              value: 'rgba(246, 246, 246, 1)'
-            }
-          ])
-        })}
-      </div>
-      <div class="w-full p-4 mb-8 bg-neutral-100 text-left text-[14px] font-bold box-border">Paragraph Example</div>
-      <div class="background-sample bg-neutral-100 ">
-        ${generateTemplate({
-          args: overrideArgs([
-            {
-              type: 'slot',
-              name: 'default',
-              value:
-                '<div class="sd-paragraph">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh justo ullamcorper odio tempor molestie phasellus dui vel id. Velit in sed non orci pellentesque vivamus nunc. At non tortor, sit neque tristique. Facilisis commodo integer hendrerit tortor. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh justo ullamcorper odio tempor molestie phasellus dui vel id. Velit in sed non orci pellentesque vivamus nunc. At non tortor, sit neque tristique. Facilisis commodo integer hendrerit tortor.</div>'
-            },
-            {
-              type: 'cssProperty',
-              name: '--gradient-color-start',
-              value: 'rgba(246, 246, 246, 0)'
-            },
-            {
-              type: 'cssProperty',
-              name: '--gradient-color-end',
-              value: 'rgba(246, 246, 246, 1)'
-            }
-          ])
-        })}
-      </div>
-    `;
+export const Inverted = {
+  name: 'Inverted',
```

Comment:
I just got aware, that you and @smfonseca added the name of the stories everywhere. This is only needed for the tests, but not here. Would be great, if we remove it in the end and maybe when you create new PRs.